qt_binder/raw_widgets.py

```diff
@@ -680,7 +680,18 @@ def add_buttons_from_context(self, context):
                 self.qobj.addButton(button, qt_id)
 
 
+# These classes are not intended to be automatically looked up from their Qt
+# classes.
+_EXCLUDE_FROM_REGISTRY = [
+    SingleChild,
+    WithLayout,
+    SpanGridLayout,
+]
```

Comment:
For the record, there is one remaining ambiguous entry in the type registry:
`QObject` maps to `[<class 'qt_binder.raw_widgets.Object'>, <class 'qt_binder.binder.NChildren'>, <class 'qt_binder.binder.Composite'>]`. I don't think it's possible to create a naked `QObject` in Designer, though.